vmware_dvs/agent/firewalls/vcenter_firewall.py

```diff
@@ -133,23 +134,35 @@ def _cleanup_removed(self):
             if current_time - remove_time > CLEANUP_REMOVE_TASKS_TIMEDELTA:
                 del self.removed[port_id]
 
+    def _cleanup_outdated_tasks(self):
+        for dvs in self.update_store:
+            self.update_store[dvs] = [item for item in self.update_store[dvs]
+                                      if item['id'] not in self.removed]
+
+    def _cleanup_network_dvs_map(self):
```

Comment:
It's not a good idea to cleanup by timer. Firewall driver has calls for remove ports. It could remove net if the last port removed only from network.